Base/usr/share/man/man5/GML-Syntax.md

```diff
@@ -1,39 +1,236 @@
 ## Name
 
-GML Basic Syntax
+GML Syntax
 
 # Description
 
-How to write GML using proper syntax.
+Overview of the GML syntax.
 
 ## Basic Syntax
 
-Each widget begins with `@GUI::`, with the name of the widget following. To define the properties of this widget, we follow with curly brackets and a list of properties.
+For a start, GML can be thought of as a QML (Qt Markup Language) derivative. JSON is used as a sub-language in some places.
+
+Each widget (or _object_, more generally) begins with `@`, with the name of the widget following. As this name refers to an actual C++ class, we need to include namespaces and separate them with `::` as usual. To define the properties of this widget, we follow the object name with curly brackets and a list of properties.
```

Comment:
GML can represent arbitrary `Core::Object`s, though. An important example are the Layout objects, which are not widgets but were specified as such in Brandon's original documentation. (I don't blame him because it can be confusing, but the distinction is important)